pkgs/tools/misc/apkeep/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchCrate, rustPlatform, openssl, pkg-config }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "apkeep";
+  version = "0.6.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-RTzYvDkmE0kgl4FSOSjDuQ5G1E0ugFU41zVAMMroofM=";
+  };
+
+  cargoSha256 = "sha256-YFs2AOMGp0WNrceK14AnigZdJl+UsQdUchpxaI7HSXw=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ];
```

Comment:
You are sure this is not using the vendored openssl?

Reply:
The build fails if I move `openssl` into `nativeBuildInputs`.